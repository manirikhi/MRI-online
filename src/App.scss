.app-container {
  background-color: #f5f5f5;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  padding-top: 8rem;
}

.app-wrapper {
  margin-top: 1rem;
  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
  width: 75%;
  background-color: white;
}