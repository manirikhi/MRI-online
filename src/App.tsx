import "./App.scss";
import OrderTable from "./components/OrderTable";

function App() {
  return (
    <main className="app-container">
      <img src="./logo.png" className="App-logo" alt="logo" />
      <div className="app-wrapper">
        <OrderTable />
      </div>
    </main>
  );
}

export default App;
