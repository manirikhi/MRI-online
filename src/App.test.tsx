import React from "react";
import { render, screen } from "@testing-library/react";
import { Provider } from "react-redux";
import App from "./App";
import { store } from "./store/store";

test("renders app", () => {
  const { asFragment } = render(
    <Provider store={store}>
      <App />
    </Provider>
  );
  const orderLabel = screen.getByText("Orders");
  expect(orderLabel).toBeInTheDocument();
  expect(asFragment()).toMatchSnapshot();
});
