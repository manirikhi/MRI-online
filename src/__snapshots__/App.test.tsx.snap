// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app 1`] = `
<DocumentFragment>
  <main
    class="app-container"
  >
    <img
      alt="logo"
      class="App-logo"
      src="./logo.png"
    />
    <div
      class="app-wrapper"
    >
      <div
        class="ag-theme-alpine order-table"
      >
        <h3
          class="order-label"
        >
          Orders
        </h3>
        <p>
          Loading orders.....
        </p>
      </div>
    </div>
  </main>
</DocumentFragment>
`;
